Update /5/ url and Deleted double import of a component (#4512)
plone/volto: packages/generator-volto/generators/app/templates/README.md

## Documentation

> A training on how to create your own website using Volto is available as part of the Plone training at [https://2022.training.plone.org/volto/index.html](https://2022.training.plone.org/volto/index.html).

Reviewer: I apologize for the last-minute change of plans. We decided that the default Volto training should be the following.

```suggestion
[Volto Hands-On](https://training.plone.org/voltohandson/index.html) is a training on how to create your own website.
```
